drivers/spi/spi-dw-core.c

```diff
@@ -246,7 +272,16 @@ static irqreturn_t dw_spi_transfer_handler(struct dw_spi *dws)
 		}
 	} else if (!dws->tx_len) {
 		dw_spi_mask_intr(dws, DW_SPI_INT_TXEI);
-		spi_finalize_current_transfer(dws->host);
+		if (dw_spi_ctlr_busy(dws)) {
+			dws->idle_wait_retries = 8;
```

Comment:
If this is an analogue for bits per data element aka "word width", SPI can be configured up to 32.

Reply:
There should be somewhere more accessible.

Reply:
And there was - `dws->n_bytes`.